modules/xmpp/strophe.jingle.js

```diff
@@ -169,32 +170,32 @@ export default class JingleConnectionPlugin extends ConnectionPlugin {
             this.sessions[sess.sid] = sess;
 
             this.eventEmitter.emit(XMPPEvents.CALL_INCOMING,
-                sess, $(iq).find('>jingle'), now);
+                sess, $_(iq, '>jingle'), now);
             break;
         }
         case 'session-accept': {
             this.eventEmitter.emit(
-                XMPPEvents.CALL_ACCEPTED, sess, $(iq).find('>jingle'));
+                XMPPEvents.CALL_ACCEPTED, sess, $_(iq, '>jingle'));
             break;
         }
         case 'content-modify': {
-            sess.modifyContents($(iq).find('>jingle'));
+            sess.modifyContents($_(iq, '>jingle'));
             break;
         }
         case 'transport-info': {
             this.eventEmitter.emit(
-                XMPPEvents.TRANSPORT_INFO, sess, $(iq).find('>jingle'));
+                XMPPEvents.TRANSPORT_INFO, sess, $_(iq, '>jingle'));
             break;
         }
         case 'session-terminate': {
             logger.log('terminating...', sess.sid);
             let reasonCondition = null;
             let reasonText = null;
 
-            if ($(iq).find('>jingle>reason').length) {
+            if ($_(iq, '>jingle>reason')) {
                 reasonCondition
-                    = $(iq).find('>jingle>reason>:first')[0].tagName;
-                reasonText = $(iq).find('>jingle>reason>text').text();
+                    = $_(iq, '>jingle>reason>:first').tagName;
+                reasonText = $_(iq, '>jingle>reason>text').textContent();
```

Comment:
`textContent` is not a function

```suggestion
                reasonText = $_(iq, '>jingle>reason>text').textContent;
```